web/react/components/post_right.jsx

```diff
@@ -289,11 +291,15 @@ module.exports = React.createClass({
         });
     },
     componentDidUpdate: function() {
-        this.resize();
+        if(!this.wasForced){
+            this.resize();
+            wasForced = false
+        }
```

Comment:
i aslo missed semicolon :). I mean in condition check `wasForced` is `false` and than `wasForced` set again to `false`